packages/mediacenter/kodi/patches/kodi-100.23-pvrmanager-early-setwakup.patch

```diff
@@ -0,0 +1,28 @@
+diff --git a/xbmc/Application.cpp b/xbmc/Application.cpp
+index b8ff91b427..b5fee706eb 100644
+--- a/xbmc/Application.cpp
++++ b/xbmc/Application.cpp
+@@ -2932,6 +2932,9 @@ void CApplication::Stop(int exitCode)
+     CLog::Log(LOGNOTICE, "stop player");
+     m_pPlayer->ClosePlayer();
+ 
++    CLog::Log(LOGNOTICE, "set wakeup");
```

Comment:
> perhaps checking if (g_PVRManager.IsStarted()) is good idea here

IMO not needed, but points to the intention. But I am wondering if g_PVRManager can ever be created from a null pointer.